src/main/scripts/org/reaktivity/specification/nukleus/tcp/control/route/client.host.and.subnet/controller.rpt

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property nukleiRG010101C ${nuklei:directory("target/nukleus-itests").controlCapacity(1024, 1024)}
+
+property controlRG010101C ${nukleiRG010101C.control("tcp")}
+
+connect "agrona://stream/bidirectional"
+        option agrona:reader ${agrona:broadcastReceiver(controlRG010101C.controller)}
+        option agrona:writer ${agrona:manyToOneWriter(controlRG010101C.nukleus)}
+
+connected
+
+write 0x00000001
+write ${controlRG010101C.nextCorrelationId()}
+write [0x01]
+write [0x06] "source"
+write 0L
+write [0x0a] "google.com"
+write 8080L
+write 0L
+write flush
+
+read 0x40000001
+read ${controlRG010101C.correlationId()}
+read (long:newClientAcceptRef)
+
+write 0x00000001
+write ${controlRG010101C.nextCorrelationId()}
+write [0x01]
+write [0x06] "source"
+write 0L
```

Comment:
I would expect this should be ${newClientAcceptRef}, assuming you are testing the ability to resolve routes based on different route extension. This is related to my comment on the other PR that you are not doing the sourceRef equality check when there's a begin extension. Once you fix that I think you'll find you need to fix this script to test the condition you are trying to force.